### PR TITLE
add PlateDetection repository with 7 different model to detect french…

### PR DESCRIPTION
**Pull request by Soso0400**

### CODE REVIEW

**Comment by BloodFutur** [comment]
Please add a description to your PR. It should briefly explain what you did in this PR. You can also show us the results you got, we will be happy to see them  :smile:

File: PlaqueDetection/test.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+if image is None:
+    print(f"Error: Unable to load image at {image_path}")
+else:
+    # Perform detection
+    results = model.predict(source=image_path, save=True)
+
+    # If you want to display the annotated results with OpenCV:
+    for result in results:
+        annotated_frame = result.plot()  # Generates an image with annotations
+        cv2.imshow("Résultats", annotated_frame)
```
Comment (BloodFutur) [inline PlaqueDetection/test.py:26]: The title is in French instead of English

File: PlaqueDetection/test.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+from ultralytics import YOLO
+import cv2
+import os
+
+# Get the absolute directory where the script is located
+script_dir = os.path.dirname(os.path.abspath(__file__))
```
Comment (BloodFutur) [inline PlaqueDetection/test.py:6]: Nice

File: PlaqueDetection/PlaqueDetection.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+    Note: The datasets/ folder in this repository will be empty. All the actual images and labels should be downloaded from Roboflow as mentioned above.
+
+# How to Run the Project
+
+    Download the dataset from Roboflow (as explained above).
+
+    Set up the environment:
+
+        - Install the required Python dependencies
```
Comment (BloodFutur) [inline PlaqueDetection/PlaqueDetection.md:43]: What are the required Python dependencies?

File: PlaqueDetection/train.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
+from ultralytics import YOLO
+import os
+
+# load YOLOv8 model 
+model = YOLO("yolov8n.yaml")  # Initialization without weights
+
+project_root = os.path.dirname(os.path.abspath(__file__))  # Current directory of the script
+data_path = os.path.join(project_root, "datasets", "data.yaml")
+# Train the model
+model.train(data=data_path, epochs=50, batch=16, imgsz=640)
```
Comment (BloodFutur) [inline PlaqueDetection/train.py:10]: How did you choose these arguments?

Comment (Soso0400) [reply PlaqueDetection/train.py:10]: It was the arguments for a basic small model, I didn't really choose them myself.

File: PlaqueDetection/test.py (hunk 1 above)
Comment (Soso0400) [reply PlaqueDetection/test.py:26]: Oh yeah, I didn't see that. Thank you

**Pull request merged**